src/backend/stream.rs

```diff
@@ -379,14 +379,14 @@ impl<'ctx> PulseStream<'ctx> {
                         s.set_state_callback(check_error, stm.as_mut() as *mut _ as *mut _);
                         s.set_write_callback(write_data, stm.as_mut() as *mut _ as *mut _);
 
-                        let buffer_size_bytes = latency_frames * stm.output_sample_spec.frame_size() as u32;
+                        let buffer_size_bytes = USecExt::to_bytes(latency_frames as u64, &stm.output_sample_spec) as u32;
```

Comment:
@kinetiknz can you please help me with this change?

Instead of getting the `frame_size` and multiply by the latency, the PA api `pa_usec_to_bytes()` is used to give this information. I would expect that those 2 methods provide the same result. However, I would like a second opinion.